surprise-pairwise-from-embedding.R

```diff
@@ -0,0 +1,151 @@
+library(lsa)

```

Comment:
You could provide additional information on what the script is about and how to install the dependencies of your script as a comment here.